panel/dist/css/chat_entry.css

```diff
@@ -0,0 +1,72 @@
+.chat-entry {
```

Comment:
## ChatInterface stretches height and introduces scrollbar

With the current version the ChatInterface suddenly stretch_both. I don't know if it is intended.

If the window is not very high it adds a scrollbar
The `_input_layout` row is also too close to the bottom of the window

![image](https://github.com/holoviz/panel/assets/42288570/65473556-9d1b-437b-9590-925a66da6dde)

This is how it looks on a higher window

![Uploading image.png…]()

Reply:
Do you have the latest PR? Mine shows this:
<img width="1261" alt="image" src="https://github.com/holoviz/panel/assets/15331990/d1c59bbf-6e49-4347-971a-bce75658cba8">